app/forms/recovery_code_form.rb

```diff
@@ -25,9 +25,9 @@ def valid_recovery_code?
     recovery_code_generator.verify(code)
   end
 
-  def result
+  def extra_analytics_attributes
     {
-      success: success,
+      method: 'recovery code',
```

Comment:
This is going to come up in 2 more PRs I have lined up. What do you think is the best way to handle this?

- leave the additional analytics attribute in the controller as it used to be
- move the attribute to the Form Object and give the key a more descriptive name like `multi_factor_auth_method`
- use a constant. If so, what would a good name be?
- something else?